src/components/Calculator.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+
+// prettier-ignore
+import {
+  Container,
+  Row,
+  Col,
+  Button,
+} from 'react-bootstrap';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      result: 0,
+    };
+  }
+
+  render() {
+    return (
+      <Container>
+        <div class="row">
+          <div class="col-lg-12">
+              this is simple div
+          </div>
+        </div>
+        <Row className="m-2">
+          <Col>
+            <div dir="rtl" className="w-100 p-2">
+              {this.state.result}
+            </div>
+          </Col>
+        </Row>
+        <Row className="m-2">
+          <Col>
+            <Button className="w-100 p-4">AC</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">+/-</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">%</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4" variant="warning">
+              /
+            </Button>
+          </Col>
+        </Row>
+        <Row className="m-2">
+          <Col>
+            <Button className="w-100 p-4">7</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">8</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">9</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4" variant="warning">
+              x
+            </Button>
+          </Col>
+        </Row>
+        <Row className="m-2">
+          <Col>
+            <Button className="w-100 p-4">4</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">5</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">6</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4" variant="warning">
+              -
+            </Button>
+          </Col>
+        </Row>
+        <Row className="m-2">
+          <Col>
+            <Button className="w-100 p-4">1</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">2</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">3</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4" variant="warning">
+              +
+            </Button>
+          </Col>
+        </Row>
+        <Row className="m-2">
+          <Col>
+            <Button className="w-100 p-4">0</Button>
+          </Col>
+          <Col xs={6}>
+            <Button className="w-100 p-4">.</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4" variant="warning">
+              =
+            </Button>
+          </Col>
+        </Row>
+      </Container>
+    );
+  }
```

Comment:
- [ ] Kindly style your calculator to loo like that of the template given. Kindly use colours similar to that used in the template.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/158475402-518760b6-117e-4ad4-925a-04646d0fa49b.png)

### Calculator Template

![image](https://user-images.githubusercontent.com/77106962/158475483-7fc9eb96-2f71-49c4-82f7-f16e7256c27b.png)